### PR TITLE
EZP-24336: As an editor, I want a proper error handling system

### PR DESCRIPTION
**Pull request by StephaneDiot**
<b>First</b>, the jira link: https://jira.ez.no/browse/EZP-24336

## Description

The goal here was to delete the "old" error message handling system, and use notifications instead. Furthermore, I added the possibility to catch error fired by sideViews.

### CODE REVIEW

**Comment by StephaneDiot** [comment]
Ready for first review : ping @dpobel @yannickroger @mhyndle

File: Resources/public/js/apps/ez-platformuiapp.js [hunk 1]
```diff
@@ -147,7 +139,6 @@ YUI.add('ez-platformuiapp', function (Y) {
 
             this.on({
                 '*:closeApp': this.close,
-                '*:fatalError': this._handleError,
                 '*:retryAction': this._retryAction,
```
Comment (dpobel) [inline Resources/public/js/apps/ez-platformuiapp.js]: you can also remove the handling of the `retryAction` event

File: Resources/public/js/apps/ez-platformuiapp.js [hunk 2]
```diff
@@ -147,7 +139,6 @@ YUI.add('ez-platformuiapp', function (Y) {
 
             this.on({
                 '*:closeApp': this.close,
```
Comment (dpobel) [inline Resources/public/js/apps/ez-platformuiapp.js]: this one can be removed as well together with the close method.

**Comment by StephaneDiot** [comment]
Updated

File: Resources/public/js/apps/ez-platformuiapp.js [hunk 3]
```diff
@@ -266,7 +234,6 @@ YUI.add('ez-platformuiapp', function (Y) {
          * @protected
          */
         _retryAction: function (retryAction) {
```
Comment (dpobel) [inline Resources/public/js/apps/ez-platformuiapp.js]: AFAIK, `_retryAction` and `close` are not used anywhere now, so they can safely be removed.

**Comment by dpobel** [comment]
+1 (when Travis agrees)

**Comment by dpobel** [comment]
the Travis failure is unrelated to the PR, you can merge

**Pull request merged**